### ISSUE TITLE
phetioFeatured overrides

### ISSUE THREAD

**Issue by arouinfar** [phetsims/beers-law-lab#305]
For #296 

Opening an issue to track the commits.

**Comment by pixelzoom** [comment]
I confirmed with @arouinfar that she can commit the overrides.js file without breaking CT, because this sim does not yet have an API file.  Ditto for concentration.

**Comment by arouinfar** [comment]
I've committed a first pass at the phetioFeatured overrides. Next steps are to review in master and cleanup as needed, and then take care of Concentration.

**Comment by arouinfar** [comment]
I reviewed the phetioFeatured overrides in BLL, found a few elements that slipped through the cracks and addressed in [a161d4b](https://github.com/phetsims/beers-law-lab/commit/a161d4ba68c395c0985c6c48ee84339a932df730). Then, I copied over all of the global and concentrationScreen overrides to Concentration (replacing beersLawLab with concentration), tested locally to make sure I didn't break anything with the replace all method. Everything looked okay, so I went ahead and committed to Concentration. Master looks fine to me, so I'm going to to close.
